src/main/java/receiver/DataValidator.java

```diff
@@ -0,0 +1,49 @@
+package receiver;
+
+import java.util.Collections;
+import java.util.List;
+
+public class DataValidator {
+	public static Float getMaxValueFromParamList(List<Float> paramList) {
+
+		if (!isValidList(paramList))
+			return 0.0F;
+
+		return Collections.max(paramList);
+	}
+
+	public static Float getMinValueFromParamList(List<Float> paramList) {
```

Comment:
When invalid, is it right that `getMinValueFromParamList` is returning a possibly valid value (`0`)?